src/pages/Filing/FilingApp/FilingErrors/FilingErrorsAlerts.tsx

```diff
@@ -15,24 +15,46 @@ function SuccessAlert({ isStep2 }: { isStep2: boolean }): JSX.Element {
   );
 }
 
-function ErrorsAlert(): JSX.Element {
+function SyntaxErrorsAlert(): JSX.Element {
   return (
     <Alert
       className='mb-[2.8125rem] [&_div]:max-w-[41.875rem] [&_p]:max-w-[41.875rem]'
-      message='Errors were found in your register'
+      message='Your register contains syntax errors'
       status='error'
     >
       There may be an issue with the data type or format of one or more values
       in your file. Make sure your register meets the requirements detailed in
       the filing instructions guide (
-      <Link target='_blank' href={dataValidationLink}>
+      <Link
+        className='border-b-[1px] border-dotted'
+        target='_blank'
+        href={dataValidationLink}
+      >
         section 4, &quot;Data validation&quot;
       </Link>
-      ) and try again. If this issue persists,{' '}
-      <Link target='_blank' href={sblHelpMail}>
-        email our support staff
+      ), make the corrections, and upload a new file.
+    </Alert>
+  );
+}
+
+function LogicErrorsAlert(): JSX.Element {
+  return (
+    <Alert
+      className='mb-[2.8125rem] [&_div]:max-w-[41.875rem] [&_p]:max-w-[41.875rem]'
+      message='Your register contains logic errors'
+      status='error'
+    >
+      There may be missing data, incorrect data, or conflicting information in
+      your file. Make sure your register meets the requirements detailed in the
+      filing instructions guide (
+      <Link
+        className='border-b-[1px] border-dotted'
+        target='_blank'
+        href={dataValidationLink}
+      >
+        section 4, &quot;Data validation&quot;
       </Link>
-      .
+      ), make the corrections, and upload a new file.
```

Comment:
I think wrapping this in a <p> tag should remove the need to apply border styles.